lib/application/widgets/theme/tags_field.dart

```diff
@@ -0,0 +1,273 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_hooks/flutter_hooks.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+import 'package:layoutr/common_layout.dart';
+import 'package:memo/application/constants/animations.dart' as anims;
+import 'package:memo/application/constants/dimensions.dart' as dimens;
+import 'package:memo/application/constants/images.dart' as images;
+import 'package:memo/application/constants/strings.dart' as strings;
+import 'package:memo/application/hooks/tags_controller_hook.dart';
+import 'package:memo/application/theme/theme_controller.dart';
+
+/// Provides the list of tags of the collection being updated.
+///
+/// These tags may be mutated by [TagsField], which the user may use to update the tags list.
```

Comment:
iMO these docs could be improved, they lack context and responsibilities

Reply:
I've improved the docs but also renamed from `TagsController` to `TagsEditingController` to follow the same naming pattern as `TextEditingController` and create the analogy that it has the same purpose but with tags. Let me know what do you think.